autoload/vital/__vital__/App/WindowSelector.vim

```diff
@@ -20,15 +21,24 @@ function! s:select(winnrs, ...) abort
           \ range(length + 1),
           \ { _, v -> get(scs, v, string(v)) },
           \)
-    let l:S = funcref('s:_statusline', [
-          \ options.statusline_hl,
-          \ options.indicator_hl,
-          \])
-    call map(keys(store), { k, v -> setwinvar(v, '&statusline', S(v, chars[k])) })
-    redrawstatus
+    if options.use_popup
+      call s:_popup(a:winnrs, options.select_chars)
+      redraw!
```

Comment:
:question:

Do you really need to use `redraw!` instead of `redraw`?

Reply:
remove `!`